src/WPF/WPF.Viewer/Samples/Data/ManageFeatures/ManageFeatures.xaml

```diff
@@ -0,0 +1,22 @@
+﻿<UserControl x:Class="ArcGIS.WPF.Samples.ManageFeatures.ManageFeatures"
+             xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+             xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+             xmlns:esri="http://schemas.esri.com/arcgis/runtime/2013">
+    <Grid>
+        <esri:MapView x:Name="MyMapView" />
+        <Border Style="{DynamicResource BorderStyle}">
+            <StackPanel>
+                <Label Content="Select a feature management operation."
+                       FontWeight="SemiBold"/>
+                <ComboBox x:Name="MethodPicker"
+                          IsEditable="False"
+                          IsEnabled="True"
+                          SelectionChanged="OnComboBoxSelectionChanged" />
+                <Label x:Name="InstructionLabel" 
+                       Content="Select an option to show instructions." />
```

Comment:
To avoid redundancy, I think this could be an empty string upon starting sample. We are already telling user to make a selection.